fastplotlib/graphics/text.py

```diff
@@ -43,11 +44,16 @@ def __init__(
             name of graphic, passed to Graphic
 
         """
+        super(TextGraphic, self).__init__(*args, **kwargs)
 
-        super(TextGraphic, self).__init__(name=name)
+        self._text = text
 
         world_object = pygfx.Text(
-            pygfx.TextGeometry(text=str(text), font_size=size, screen_space=False),
+            pygfx.TextGeometry(text=str(text),
+                               font_size=size,
+                               screen_space=True,
+                               anchor="middle-center"
+                               ),
             pygfx.TextMaterial(
```

Comment:
any other useful kwargs to have for material?

Reply:
no, I don't think so...not anything that should be exposed to a user